### ISSUE TITLE
Small question regarding Throttle. 

### ISSUE THREAD

**Issue by Dreams-Visions** [domdavis/ch-products-elite-map#10]
Warmest greetings. Me again with what is hopefully another simple question.

By default in this map, the throttle range set all the way back is 0 and all the way forward is maximum throttle, and a button has to be toggled to allow the axis to function is reverse thrust. I plan on adding a DIY notch so that the throttle will have a bump when it passes center. What I'd like is to set it so that:

- 0 speed is when the throttle is resting to the middle 
- Pushing the stick forward increases the speed linearly, hitting 100% throttle when all the way forward 
- Pushing the stick below the middle increases reverse speed linearly, hitting -100% reverse throttle when all the way back.

it would look like this if looking at your Pro Throttle from above:

~~~~
=======
   |
 100%
   |
   |
   |
 0%
   |
   |
   |
-100%
   |
=======
~~~~

I'm sure this is a very standard configuration but being still very green with regards to CMS scripting, I'm not entirely sure how to tell the software to execute this. I've tried for a few hours and I had to admit that I'm a bit over-matched. If you have a moment to let me know what to revise in the CMS to get this behavior, I'd be very grateful. Thank you again for the great map. 500 hours of ED in, I can't play this game without it.

**Comment by domdavis** [comment]
The reason you're being overmatched is this is actually an in game setting ;)

I can't remember what it's called off the top of my head, but if you take the map with no changes, go into the game settings there is one that deals with how the throttle behaves. Change that and you'll get the desired behaviour. When I'm in front of the game next I'll confirm the name of the setting.

**Comment by Dreams-Visions** [comment]
Only mildly embarrassing. :D 

Do let me know if you happen up that setting. tyty

**Comment by domdavis** [comment]
Change Throttle Axis Range from Forward only to Full Range.

**Comment by Dreams-Visions** [comment]
Bless you sir. Sorry for the once-a-month question. I really do appreciate your help. Up to a battle conda + FdL + battle Python + Courier and I've yet to get bored thanks in large part to your efforts on this map. Ty again. Do you have any plans to convert or otherwise develop a similar map for Star Citizen one day? I'm sure there would be lots of grateful CH Products owners! :dancer:

**Comment by domdavis** [comment]
I've put Elite down for a couple of weeks due to playing other things and family commitments. Hoping to get back into it again soon. While I'm going through an off period I'll take another look at Star Citizen. I fully intend to make a map for that, but there wasn't enough of a game there last time I looked - think I got as far as jotting down the controls and working out what they do. Once more gameplay comes out you'll start seeing basic maps coming out :)
